ES/ESmy016.xml

```diff
@@ -31,325 +31,126 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
+                              ዕቀበኒ፡ ለመልአከ፡ ኀጕል፡ ከመ፡ ኢይኩን፡ መርከቦ፨<foreign xml:lang="en">
+                                 <locus target="#33v"/>Salutation to the reminding of
+                                 your name, the name of wonder and astonishment. And salutation to your
+                                 head, which the light of the divinity covers. St Sabbath of Christians,
+                                 repose of the sinners from the torments. Protect me so that I would not
+                                 become a ship of the angel of perdition!</foreign>
+                           </incipit>
+                           <explicit xml:lang="gez">
+                              <locus target="#44r"/><hi rend="rubric">ሰን፡ ክ፡ ቅ፡</hi> ሀብኒ፡ ፍትሐ፡ ሣህልኪ፡
+                              ለላዕለቴ: በከመ፡ ይነሥእ፡ ዓሰባ፡ ፃማሁ፡ ዝንተ፡ አሳባ፡ ድርሳንየ፡ ኢይጥፋዕ፡ በከንቱ።<foreign xml:lang="en">
+                                 <locus target="#44r"/>St Sabbath of Christians, give
+                                 me the justice of your high compassion, as he takes a reward
+                                 of his toils, this the reward of my writing may not perish in
+                                 vain.</foreign>
+                           </explicit>
+                           </msItem>
+                                 </msContents>
                <physDesc>
                   <objectDesc form="Codex">
                      <supportDesc>
                         <support>
-                           <material key="parchment"/>
+                           <material key="parchment">Parchment leaves are poorly cut, with uneven edges.
+                           Cuts or holes are crudely amended on <locus target="#34 #35 #44"/>.</material>
                         </support>
-                        <extent>Made of <measure unit="leaf">44.0</measure> folios, of which
-                              <measure unit="leaf" type="blank"/> blank 42rv, 44v, in <measure unit="quire">6.0</measure> quires . <dimensions type="outer">
+                        <extent>
+                          <measure unit="leaf">44</measure>
+                        <measure unit="leaf" type="blank">3</measure>
+                        <locus target="#42r #42v #44v"/>
+                        <measure unit="quire">6</measure>
+                        <dimensions type="outer">
                               <height unit="mm">145</height>
                               <width unit="mm">110</width>
                               <depth unit="mm">30</depth>
                            </dimensions>
                         </extent>
-                        <foliation><!--2016-10-18+02:00 Updated MY-016 from DomLib--></foliation>
+                        <foliation>Foliation by Ethio-SPaRE, in pencil, in the outer corner of the bottom margin.</foliation>
                         <collation>
                            <list>
-                              <item xml:id="q1">
```

Comment:
quire descriptions are missing now, they can be left. in particular they must be there in case of single leaves and other anomalies, otherwise the app won't work

Reply:
done